CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Master (Unreleased)
 
+* **(Breaking)** Drop Ruby 2.5 support. ([@ydah][])
```

Comment:
I wouldn’t call it a breaking change (which would force us to do a major version release). Because of the `required_ruby_version` in our gemspec, those of our users that are still using Ruby 2.5 are simply unable to install the new version – and nothing will break for them.

Reply:
I thought that was indeed true, so I deleted it. Thank you so much.